test/unit/network_spec.js

```diff
@@ -94,7 +94,7 @@ describe('network', function() {
         rangeChunkSize: 65536,
         disableStream: false,
       },
-      disableRange: false
+      disableRange: true
```

Comment:
Why is this change made? It looks unrelated to the patch. The same goes for the other changes in this file. Could you explain this a bit?